NEWS

```diff
@@ -66,6 +66,17 @@ New Functionality
   with CD being populated from requests and AD being populated from replies,
   unless there wasn't a reply.
 
+- Add new configure option ``--disable-port-prealloc``. Prior to the merging of
+  the Windows port, Zeek preallocated PortVal objects for every valid port
+  number. This came at a cost of a minor amount of memory, but provided better
+  performance. With the Windows port, we removed this preallocation in favor of
+  the smaller memory footprint at the cost of having to allocate a new object
+  each time a new port number is seen in network traffic plus a potentially
+  slower lookup in a map vs an array. Over the span of a long run of Zeek, this
+  memory savings is lost but the performance cost still exists. This change
```

Comment:
Nit pickery. Maybe start the paragraph with: Revert to pre-allocated PortVals (or additionally put under "Changed Functionality"). Seems more relevant than the new config option and is a bit hidden at the end of the paragraph.

I'm also wondering if this is 5.2.1 material, or if we should suggest to distributors to pick the patch for better performance.